<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form1</class>
 <widget class="QWidget" name="Form1">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>问答对自动生成器</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>600</width>
     <height>280</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>400</width>
      <height>70</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_apikey">
       <property name="placeholderText">
        <string>sk-xxx</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_proxy_pass">
       <property name="text">
        <string>代理地址（选填）</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_proxy_pass">
       <property name="placeholderText">
        <string>https://api.openai.com/</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_apikey">
       <property name="text">
        <string>ChatGPT ApiKey</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_proxy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>400</width>
      <height>110</height>
     </rect>
    </property>
    <property name="title">
     <string>代理服务器（仅支持HTTP模式）</string>
    </property>
    <widget class="QCheckBox" name="checkBox_proxy">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>启用代理</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_proxy" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>55</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_proxy_addr">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>地址：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_proxy_addr">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>5</y>
        <width>175</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_proxy_port">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>端口：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_proxy_port">
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>5</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>8080</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_hint">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>570</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>注意：本程序为开源应用，本身不存在任何后门或ApiKey泄露风险！谨防欺诈代理！</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>打赏作者</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_buttons">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_test">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>测试连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>下一步</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
